tidb-lightning/tidb-lightning-physical-import-mode-usage.md

```diff
@@ -126,11 +126,9 @@ mysql> select table_name,index_name,key_data,row_data from conflict_error_v1 lim
 
 根据上述信息人工甄别需要保留的重复数据，手动插回原表即可。
 
-## 导入数据到生产集群
+## 导入时限制调度范围从集群降低到表级别
```

Comment:
If this title is changed, the user doc links in 6.2.0 release notes should be updated as well.